samples/AllControls/Droid/AllControls.Droid.fsproj

```
@@ -163,10 +164,13 @@
     </Reference>
     <Reference Include="Xamarin.Forms.Xaml">
       <HintPath>../../../packages/androidapp/Xamarin.Forms/lib/MonoAndroid10/Xamarin.Forms.Xaml.dll</HintPath>
-    </Reference>
+    </Reference> 
     <Reference Include="FSharp.Core">
       <HintPath>../../../packages/androidapp/FSharp.Core/lib/netstandard1.6/FSharp.Core.dll</HintPath>
     </Reference>
+    <Reference Include="FSharp.Data">
+      <HintPath>../../../packages/androidapp/FSharp.Data.3.0.0/lib/netstandard2.0/FSharp.Data.dll</HintPath>
```

Paket doesn't add the version number in the path
```suggestion
      <HintPath>../../../packages/androidapp/FSharp.Data/lib/netstandard2.0/FSharp.Data.dll</HintPath>
```